### DISCUSSION TITLE
My flashforge goes to sleep on wifi

### DISCUSSION THREAD

**Discussion by patleith** [xblax/flashforge_ad5m_klipper_mod#110] [Q&A]
Hi, when i use the wifi, after x amount of time, it look like the wifi goes to sleep and i cant access the printer at all,if i touch the screen i can see that the printer is on, but the wifi does not reconnect or stay connected
and  i have to reboot manualy the printer from the switch on the back , but if i use the network cable, it does not seem to go to sleep and as of now (after 3 hours of not doing anything it still connect 

is there a parameters i can check or change for the sleep on the wifi ?

thanks in advance

**Comment by consp** [comment]
> is there a parameters i can check or change for the sleep on the wifi ?

There is no sleep setting in iwd afaik. Could you provide the information in `/var/log/messages`?

For what it's worth: I have had the printer on for hours and never had this issue, connected only to wifi.

**Comment by xblax** [comment]
> For what it's worth: I have had the printer on for hours and never had this issue, connected only to wifi.

Same for me, I even had the printer running over weekend once. As a test I just disabled the Wifi of my router and turned it back on to make sure the printer reconnects. That seems to work as well.
